tachyon/math/finite_fields/generator/generator_util.h

```diff
@@ -30,7 +30,7 @@ std::string MpzClassToMontString(const mpz_class& v_in, const mpz_class& m_in) {
     gmp::CopyLimbs(v_in, v.limbs);
   }
 
-  BigInt<N * 2> mul_result = v.Mul(r2);
+  BigInt<N * 2> mul_result = v.MulExtend(r2);
```

Comment:
To standardize <2 * N>
```suggestion
  BigInt<2 * N> mul_result = v.MulExtend(r2);
```
Though there are separate instances of 2 * N and N * 2, personally, 2 * N seems better.